### ISSUE TITLE
Disable "Please balance team" messages

### ISSUE THREAD

**Issue by def-** [kaitlynia/xpanic#15]
Is it possible server side?

**Comment by kaitlynia** [comment]
I don't think this can be fixed without modifying the clients in use, since that's where it's determined. Review [this line of DDNet code](https://github.com/ddnet/ddnet/blob/master/src/game/client/components/hud.cpp#L869) as an example. We could give the client incorrect team sizes, so when [the difference is calculated](https://github.com/ddnet/ddnet/blob/master/src/game/client/components/hud.cpp#L599) it doesn't show the message, but I'm not sure if this is a good solution.

**Comment by kaitlynia** [comment]
Ah, it's lumped into the `pInfo` logic on the client anyway, where your team, number of players, and number of players in each team is determined. If we just didn't send the `NETOBJTYPE_PLAYERINFO` type I think it would break other things.
